kafka-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/kafka/KafkaProperties.java

```diff
@@ -54,8 +54,10 @@
 @ConfigurationProperties(prefix = "axon.kafka")
 public class KafkaProperties {
 
+    public static final String DEFAULT_TOPIC = "local.event";
```

Comment:
As a public constant, it should contain some JavaDoc. Or we make this private, which should be fine too if you ask me.